examples/spring-boot/build.gradle

```diff
@@ -0,0 +1,23 @@
+buildscript {
```

Comment:
just:
```groovy
plugins {
	id 'org.springframework.boot' version '2.1.2.RELEASE'
}
```

Reply:
That's what https://start.spring.io/ generates by the way :wink:

Reply:
You may want o open an issue for the Spring Start project. The plugins DSL is the preferred syntax.

Reply:
Looks like the issue already exists: https://github.com/spring-io/initializr/issues/692